pandas/tests/arithmetic/test_numeric.py

```diff
@@ -833,6 +833,15 @@ def test_ufunc_coercions(self, holder):
         exp = tm.box_expected(exp, box)
         tm.assert_equal(result, exp)
 
+    @pytest.mark.parametrize('holder', [pd.Int64Index, pd.UInt64Index,
```

Comment:
holder -> box

Reply:
The tests above also use holder, so I wanted to be consistent, but can change them all

Reply:
yes, we have settled on box a while back